src/Stryker.Core/Stryker.Core/DiffProviders/GitDiffProvider.cs

```diff
@@ -36,7 +36,7 @@ public DiffResult ScanDiff()
 
             if (commit == null)
             {
-                throw new InputException("Could not determine a commit to check for diff. Please check you have provided the correct value for --git-source");
+                throw new InputException("Could not determine a commit to check for diff. Please check you have provided the correct source.");
```

Comment:
```suggestion
                throw new InputException("Could not find a commit to diff. Please check you have provided the correct committish for 'since'.");
```